drivers/SmartThings/philips-hue/src/lunchbox/rest.lua

```diff
@@ -129,6 +129,9 @@ local function parse_chunked_response(original_response, sock)
 end
 
 local function handle_response(sock)
+  if require("version").api > 7 then
```

Comment:
I think this, but need to double check:
```suggestion
  if require("version").api > 8 then
```